crypto/err/openssl.txt

```diff
@@ -689,6 +689,7 @@ EVP_R_INVALID_KEY:163:invalid key
 EVP_R_INVALID_KEY_LENGTH:130:invalid key length
 EVP_R_INVALID_LENGTH:221:invalid length
 EVP_R_INVALID_NULL_ALGORITHM:218:invalid null algorithm
+EVP_R_INVALID_NULL_ARGUMENT:226:invalid null argument
```

Comment:
Please use `ERR_R_PASSED_NULL_PARAMETER`

Reply:
I thought there was an error for this but didn't find it.  Have changed.